src/mame/drivers/at.cpp

```diff
@@ -1416,6 +1418,23 @@ ROM_START( comportiii )
 	ROMX_LOAD( "109737-002.bin", 0x10001, 0x8000, CRC(8463cc41) SHA1(cb9801591e4a2cd13bbcc40739c9e675ba84c079), ROM_SKIP(1) | ROM_BIOS(1) )
 ROM_END
 
+// Compaq SLT/286 - complains about "102 - System board failure"
+ROM_START( comslt286 )
+   ROM_REGION(0x20000,"bios", 0)
+   	ROM_LOAD16_BYTE( "compaq_slt286-bios-revision_ j.2-even.bin", 0x10000, 0x8000, CRC(77e894e0) SHA1(e935e62e203ec67eaab198c15a36cc0078fd35b0))
```

Comment:
The space in the filename should probably be removed.

Reply:
Thanks, I'll correct that with the next part of the PR